Documentation/ApiOverview/Routing/_AdvancedRoutingConfiguration/_ext_localconf.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+use MyVendor\MyPackage\Routing\CustomEnhancer;
```

Comment:
```suggestion
use MyVendor\MyExtension\Routing\CustomEnhancer;
```

As the caption refers to "my_extension".